src/init.cpp

```diff
@@ -1865,6 +1866,19 @@ bool AppInitMain(NodeContext& node, interfaces::BlockAndHeaderTipInfo* tip_info)
 
     connOptions.m_i2p_accept_incoming = args.GetBoolArg("-i2pacceptincoming", DEFAULT_I2P_ACCEPT_INCOMING);
 
+    if (args.GetBoolArg("-sensitiverelayowntx", DEFAULT_SENSITIVE_RELAY_OWN_TX)) {
```

Comment:
In e310a0d285db0b82b0c201a56f92a7bfe4663226, you added these verifications, I think it would be useful if you add coverage for them in b6fd6b525d3ee4d89331dc04d86f854cb1decccd

Suggestion (did it to test e310a0d285db0b82b0c201a56f92a7bfe4663226):
```diff
diff --git a/test/functional/p2p_local_tx_relay.py b/test/functional/p2p_local_tx_relay.py
index 0b115352d..46215110e 100755
--- a/test/functional/p2p_local_tx_relay.py
+++ b/test/functional/p2p_local_tx_relay.py
@@ -140,12 +140,12 @@ class P2PLocalTxRelay(BitcoinTestFramework):

     def setup_nodes(self):
         # Start a SOCKS5 proxy server.
-        socks5_server_config = Socks5Configuration()
-        socks5_server_config.addr = ("127.0.0.1", p2p_port(new_p2p_index()))
-        socks5_server_config.unauth = True
-        socks5_server_config.auth = True
+        self.socks5_server_config = Socks5Configuration()
+        self.socks5_server_config.addr = ("127.0.0.1", p2p_port(new_p2p_index()))
+        self.socks5_server_config.unauth = True
+        self.socks5_server_config.auth = True

-        self.socks5_server = Socks5Server(socks5_server_config)
+        self.socks5_server = Socks5Server(self.socks5_server_config)
         self.socks5_server.start()

         ports_base = p2p_port(MAX_NODES) + 15000
@@ -171,7 +171,7 @@ class P2PLocalTxRelay(BitcoinTestFramework):
             [
                 "-peerbloomfilters", # needed to test replies to MEMPOOL
                 "-sensitiverelayowntx",
-                f"-proxy={socks5_server_config.addr[0]}:{socks5_server_config.addr[1]}"
+                f"-proxy={self.socks5_server_config.addr[0]}:{self.socks5_server_config.addr[1]}"
             ]
         ])
         self.start_nodes()
@@ -243,6 +243,15 @@ class P2PLocalTxRelay(BitcoinTestFramework):
         self.log.info("Waiting for normal broadcast to another observer")
         observer_outbound.wait_for_inv([inv])

+        self.log.info("Test an invalid usage of -sensitiverelayowntx throw an init error")
+        self.stop_node(0)
+        args_errors = {
+            "Sensitive relay of own transactions requested (-sensitiverelayowntx), but none of Tor or I2P networks
is reachable": ["-sensitiverelayowntx"],
+            "Sensitive relay of own transactions requested (-sensitiverelayowntx), but -connect is also configured.
 They are incompatible because the sensitive relay needs to open a new connection to a randomly chosen Tor or I2P pe
er" : ["-sensitiverelayowntx", "-connect='127.0.0.1'", f"-proxy={self.socks5_server_config.addr[0]}:{self.socks5_ser
ver_config.addr[1]}"]
+        }
+        for msg, args in args_errors.items():
+            node0.assert_start_raises_init_error(extra_args=args, expected_msg=f'Error: {msg}')
+

 if __name__ == '__main__':
     P2PLocalTxRelay().main()
```

Reply:
Added this test, but in `feature_config_args.py` because it checks interaction between some config args. That fits better than `p2p_local_tx_relay.py`:
> Test how locally submitted transactions are sent to the network when sensitive relay is used.